### PR TITLE
Update to Vanilla v1.8

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- Updated version of site to v1.8
- Updated sub-heading in hero to `h4` and removed is-deep class
- Fixed navigation logo
- Add favicon to site

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8007](http://0.0.0.0:8007)
4. Compare with [live](https://docs.ubuntu.com/) and ensure that the site looks the same (or better)

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -3,15 +3,18 @@
...
           </g>
         </svg>
-        documentation
+        <h4 style="padding: 0.5rem 0.5rem; margin-bottom: -10px;">
```
Comment (deadlight) [inline templates/index.html]: update to 0.8rem

File: templates/index.html [hunk 2]
```diff
@@ -3,15 +3,18 @@
...
+          <span class="u-hide--small">documentation</span>
+          <span class="u-hide--medium u-hide--large" style="line-height: 2rem; font-size: 1.30612rem;">developer</span>
```
Comment (deadlight) [inline templates/index.html]: update text

File: templates/index.html (hunk 1 above)
Comment (kwm14) [reply templates/index.html]: Done 👍

File: templates/index.html (hunk 2 above)
Comment (kwm14) [reply templates/index.html]: Done 👍

**Comment by kwm14** [comment]
@deadlight updated your changes. Ready for another review 👍

**Comment by deadlight** [comment]
It doesn't look like this is working correctly. I'll take a look and try and find the correct value.

**Comment by kwm14** [comment]
Ready for re-review @deadlight 😄

**Pull request merged**